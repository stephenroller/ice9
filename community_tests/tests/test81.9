# mult and logical and										succeed

write 2 * 2;
write 2 * 4 * 7 * 9;
write 2 * (3 * 4);
write ? (true * false);
write ? (true * (false * false));
write ? ((true * false) * (false * false));
