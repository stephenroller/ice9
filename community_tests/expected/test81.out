Loading file: generated/test81.tm
4 
504 
24 
0 
0 
0 
Number of instructions executed = 120
